system/forms/formbuilder/item-types/page.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form i18nBaseUri="formbuilder.item-types.page:" feature="formbuilder">
+	<tab id="default" sortorder="10">
+		<fieldset id="default" sortorder="10">
+			<field name="label"     sortorder="10" control="textInput" required="true" />
+			<field name="condition" sortorder="20" control="conditionPicker" rulecontext="formbuilderSubmission" quickEdit="true" quickAdd="true" />
```

Comment:
I think the context here should be `webRequest` which _includes_ the `formbuilderSubmission` context. This way, you get the form builder submission expressions but can also create conditions on other factors such as whether or not the user is already a member, etc.